.travis.yml

```diff
@@ -22,6 +23,7 @@ jobs:
     - scripts/setup-minikube-for-linux.sh
     - make test CHARTS=enterprise-suite-latest
   - name: Minikube - Frontend
+    node_js: 9.4.0
```

Comment:
Wondering if that should be
`node_js: "9.4.0"`
based on the doco I see.  Not sure it makes a difference though.